js/src/css/shared/_woocommerce-admin.scss

```diff
@@ -77,10 +77,6 @@
 			max-width: wpVariables.$grid-unit-60;
 		}
 	}
-
-	.components-notice:not(.app-notice) {
```

Comment:
@iamdharmesh Removing this can cause some regressions. Any reasons why we are removing it?

Reply:
I had removed it because it was overriding the bottom margin. Reverted this now.